src/config/config.js

```diff
@@ -52,6 +52,8 @@ const Config = {
   logLevel: (forceLogLevel && forceLogLevel[1]) || env.REACT_APP_LOG_LEVEL || 'debug',
   serverUrl: env.REACT_APP_SERVER_URL || 'http://localhost:3003',
   gunPublicUrl: env.REACT_APP_GUN_PUBLIC_URL || 'http://localhost:3003/gun',
+  nftStorageKey: env.REACT_APP_NFT_STORAGE_KEY,
+  nftPeers: (env.REACT_APP_NFT_PEERS || "https://ipfs.io/ipfs/{cid},https://{cid}.ipfs.dweb.link").split(","),
```

Comment:
```suggestion
  nftPeers: (env.REACT_APP_NFT_PEERS.split(',') || "https://ipfs.io/ipfs/{cid},https://{cid}.ipfs.dweb.link").split(","),
```
if you split the default string you need to split the env var as well but you should probably check it is not falsy first

Reply:
it splits the OR result

Reply:
.split returns array and could be called from the string.
so your suggestion a) will fall if env var is undefined b) will return an empty array if it's an empty string because even an array is empty it casts as true

with my implementation everything is ok

(undefined || defaultStringValue).split() => [default, value, by, chunks]
('' || defaultStringValue).split() => [default, value, by, chunks]
('peer1,peer2' || defaultStringValue).split() => [peer1, peer2]